oxidation/libparsec/crates/protocol/schema/authenticated_cmds/pki_enrollment_accept.json

```diff
@@ -0,0 +1,52 @@
+[
+    {
+        "label": "PkiEnrollmentAccept",
+        "major_versions": [
+            2,
+            3
+        ],
+        "req": {
+            "cmd": "pki_enrollment_accept",
+            "other_fields": [
+                {
+                    "name": "accept_payload",
+                    "type": "Bytes"
+                },
+                {
+                    "name": "accept_payload_signature",
+                    "type": "Bytes"
+                },
+                {
+                    "name": "accepter_der_x509_certificate",
+                    "type": "Bytes"
+                },
+                {
+                    "name": "device_certificate",
+                    "type": "Bytes"
+                },
+                {
+                    "name": "enrollment_id",
+                    "type": "UUID"
+                },
+                {
+                    "name": "redacted_device_certificate",
```

Comment:
Should keep the comment from the python schema:
```
// Same certificates than above, but expurged of human_handle/device_label
```

Reply:
Also the order of fields should be kept from the python schema (so `user_certificate` & `device_certificate`, then the redacted ones) to keep the comment's meaning ^^